### PR TITLE
Fix headers and variable names to meet coding style standards

### PR DESCRIPTION
**Pull request by errissa**
Files in Rendering and Rendering/Filament were examined and fixed

---
This change is [Reviewable](https://reviewable.io/reviews/intel-isl/open3d/1842)

### CODE REVIEW

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5535289&bid=18144056)

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5535289&bid=18144056)

File: src/Open3D/Visualization/Rendering/Filament/FilamentView.h [hunk 1]
```diff
@@ -26,12 +26,12 @@
 
 #pragma once
 
-#include "Open3D/Visualization/Rendering/View.h"
+#include <filament/Color.h>
```
Comment (pvarvak) [inline src/Open3D/Visualization/Rendering/Filament/FilamentView.h:30]: delete blank line?

File: src/Open3D/Visualization/Rendering/Filament/PointCloudBuffers.cpp [hunk 2]
```diff
@@ -24,19 +24,18 @@
...
-#include "FilamentEngine.h"
-#include "FilamentResourceManager.h"
-#include "Open3D/Geometry/BoundingVolume.h"
-#include "Open3D/Geometry/PointCloud.h"
-
-#include <Eigen/Core>
-
 #include <filament/IndexBuffer.h>
 #include <filament/VertexBuffer.h>
 #include <filament/geometry/SurfaceOrientation.h>
```
Comment (pvarvak) [inline src/Open3D/Visualization/Rendering/Filament/PointCloudBuffers.cpp:30]: delete empty line?

File: src/Open3D/Visualization/Rendering/Filament/TriangleMeshBuffers.cpp [hunk 3]
```diff
@@ -41,6 +34,12 @@
```
Comment (pvarvak) [inline src/Open3D/Visualization/Rendering/Filament/TriangleMeshBuffers.cpp:34]: delete empty line?

File: src/Open3D/Visualization/Rendering/Filament/TriangleMeshBuffers.cpp [hunk 4]
```diff
@@ -364,108 +364,107 @@ RenderableManager::PrimitiveType TriangleMeshBuffersBuilder::GetPrimitiveType()
...
 
     size_t stride = sizeof(BaseVertex);
     VertexBuffer* vbuf = nullptr;
-    if (hasUVs) {
+    if (has_uvs) {
         stride = sizeof(TexturedVertex);
-    } else if (hasColors) {
+    } else {
         stride = sizeof(ColoredVertex);
     }
```
Comment (pvarvak) [inline src/Open3D/Visualization/Rendering/Filament/TriangleMeshBuffers.cpp:423]: can we move setting up stride to be together with calls to CreateTexturedBuffers, CreateColoredBuffers, and CreatePlainBuffers, above, so we can guarantee they always match?

File: src/Open3D/Visualization/Rendering/Renderer.cpp [hunk 5]
```diff
@@ -24,26 +24,25 @@
...
 
 #include "Open3D/Geometry/Image.h"
 #include "Open3D/Utility/Console.h"
+#include "Open3D/Visualization/Rendering/RenderToBuffer.h"
 
 namespace open3d {
 namespace visualization {
 
-static ResourceLoadRequest::ErrorCallback sDefaultErrorHandler =
+static ResourceLoadRequest::ErrorCallback kDefaultErrorHandler =
```
Comment (pvarvak) [inline src/Open3D/Visualization/Rendering/Renderer.cpp:36]: if we are going to prefix with "k", lets make this "static const"?

File: src/Open3D/Visualization/Rendering/RendererStructs.h [hunk 6]
```diff
@@ -26,11 +26,12 @@
 
 #pragma once
 
-#include "RendererHandle.h"
-
 #include <json/json.h>
+
```
Comment (pvarvak) [inline src/Open3D/Visualization/Rendering/RendererStructs.h:30]: delete empty line?

File: src/Open3D/Visualization/Rendering/Filament/FilamentGeometryBuffersBuilder.h [hunk 7]
```diff
@@ -26,14 +26,16 @@
...
 
+// NOTE: This header must precede the Filament headers otherwise a conflict
+// occurs between Filament and standard headers
 #include "Open3D/Visualization/Rendering/RendererHandle.h"
 
-#include <memory>
-#include <tuple>
-
 #include <filament/Box.h>
 #include <filament/RenderableManager.h>
```
[the remark this answers is not in the document]
Comment (errissa) [reply src/Open3D/Visualization/Rendering/Filament/FilamentGeometryBuffersBuilder.h:35]: clang-format inserted the blank line. check-style fails without it.

File: src/Open3D/Visualization/Rendering/Filament/FilamentView.h (hunk 1 above)
Comment (errissa) [reply src/Open3D/Visualization/Rendering/Filament/FilamentView.h:30]: clang-format inserted the blank line. check-style fails without it.

File: src/Open3D/Visualization/Rendering/Filament/PointCloudBuffers.cpp (hunk 2 above)
Comment (errissa) [reply src/Open3D/Visualization/Rendering/Filament/PointCloudBuffers.cpp:30]: clang-format inserted the blank line. check-style fails without it.

File: src/Open3D/Visualization/Rendering/Filament/TriangleMeshBuffers.cpp (hunk 3 above)
Comment (errissa) [reply src/Open3D/Visualization/Rendering/Filament/TriangleMeshBuffers.cpp:34]: clang-format inserted the blank line. check-style fails without it.

File: src/Open3D/Visualization/Rendering/Renderer.cpp (hunk 5 above)
Comment (errissa) [reply src/Open3D/Visualization/Rendering/Renderer.cpp:36]: I wasn't 100% sure how to handle this when reviewing the code. However, I interpreted the Google style guide to suggest that static, file scope variables should have the 'k' prefix: "All such variables with static storage duration (i.e. statics and globals, see Storage Duration for details) should be named this way. " I have no objection to changing to static const assuming DefaultErrorHandler is never changed. I'll have to double check that.

File: src/Open3D/Visualization/Rendering/RendererStructs.h (hunk 6 above)
Comment (errissa) [reply src/Open3D/Visualization/Rendering/RendererStructs.h:30]: inserted by clang-format. (see above)

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5535289&bid=18144056)

File: src/Open3D/Visualization/Rendering/Filament/TriangleMeshBuffers.cpp (hunk 4 above)
Comment (errissa) [reply src/Open3D/Visualization/Rendering/Filament/TriangleMeshBuffers.cpp]: changed.

File: src/Open3D/Visualization/Rendering/Filament/FilamentGeometryBuffersBuilder.h (hunk 7 above)
[the remark this answers is not in the document]
Comment (errissa) [reply src/Open3D/Visualization/Rendering/Filament/FilamentGeometryBuffersBuilder.h]: this turns out to be a difference between the way clang-format-5 and clang-format-9 implement SortIncludes. Empty line removed.

File: src/Open3D/Visualization/Rendering/Filament/PointCloudBuffers.cpp (hunk 2 above)
Comment (errissa) [reply src/Open3D/Visualization/Rendering/Filament/PointCloudBuffers.cpp]: this turns out to be a difference between the way clang-format-5 and clang-format-9 implement SortIncludes. Empty line removed.

File: src/Open3D/Visualization/Rendering/Filament/TriangleMeshBuffers.cpp (hunk 3 above)
Comment (errissa) [reply src/Open3D/Visualization/Rendering/Filament/TriangleMeshBuffers.cpp:34]: this turns out to be a difference between the way clang-format-5 and clang-format-9 implement SortIncludes. Empty line removed.

File: src/Open3D/Visualization/Rendering/Filament/FilamentView.h (hunk 1 above)
Comment (errissa) [reply src/Open3D/Visualization/Rendering/Filament/FilamentView.h]: this turns out to be a difference between the way clang-format-5 and clang-format-9 implement SortIncludes. Empty line removed.

File: src/Open3D/Visualization/Rendering/RendererStructs.h (hunk 6 above)
Comment (errissa) [reply src/Open3D/Visualization/Rendering/RendererStructs.h]: this turns out to be a difference between the way clang-format-5 and clang-format-9 implement SortIncludes. Empty line removed.

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5535289&bid=18144056)

File: src/Open3D/Visualization/Rendering/Renderer.cpp (hunk 5 above)
Comment (errissa) [reply src/Open3D/Visualization/Rendering/Renderer.cpp]: changed to static const

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5535289&bid=18144056)

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5535289&bid=18144056)

**Pull request merged**